frontend/src/metabase/actions/containers/ActionCreator/FormCreator/FormCreator.styled.tsx

```diff
@@ -40,7 +40,10 @@ export const ExplainerTitle = styled.h3`
   margin-bottom: ${space(1)};
 `;
 
-export const ExplainerText = styled.p`
+// REMINDER: mention that using `styled.p` causes a
```

Comment:
What do y'all think? I think we should try to keep our HTML semantically useful but having the `p > ul` is causing errors, so I'm not sure we should keep this as a `p`